src/pages/api/restaurant-finder.ts

```diff
@@ -16,6 +17,8 @@ interface RestaurantFinderRequest extends NextApiRequest {
     body: RestaurantFinderQuery;
 }
 
+const GEOAPIFY_API_KEY: string | undefined = process.env.GEOAPIFY_KEY;
```

Comment:
im pretty sure to use this u need

```
import 'dotenv/config'
```

Reply:
It works tho, plus this is ur code I copy pasted here 😒